packages/roosterjs-react-command-bar/lib/plugins/RoosterCommandBarPlugin.ts

```diff
@@ -14,24 +17,56 @@ export default class RoosterCommandBarPlugin implements EditorPlugin, RoosterCom
 
     private editor: Editor;
     private commandBars: RoosterCommandBar[] = [];
+    private defaultShortcut: DefaultShortcut = new DefaultShortcut();
+    private strings: { [key: string]: string };
+
+    constructor(strings?: { [key: string]: string }) {
+        this.strings = strings;
+    }
 
     public initialize(editor: Editor): void {
+        this.defaultShortcut.initialize(editor);
         this.editor = editor;
     }
 
     public dispose(): void {
         if (this.editor) {
             this.editor = null;
         }
+
         if (this.commandBars.length > 0) {
-            this.commandBars.forEach((_, i) => this.commandBars[i] = undefined);
+            this.commandBars.forEach((_, i) => (this.commandBars[i] = undefined));
             this.commandBars = [];
         }
+
+        if (this.defaultShortcut) {
+            this.defaultShortcut.dispose();
+            this.defaultShortcut = null;
+        }
     }
 
     public onPluginEvent(event: PluginEvent): void {
         if (this.commandBars && RoosterCommandBarPlugin.EventTypesToHandle[event.eventType]) {
             this.commandBars.forEach(commandBar => commandBar.refreshFormatState());
+            return;
+        }
+
+        if (event.eventType === PluginEventType.KeyDown) {
+            this.handleShortcuts(event);
+        }
+    }
+
+    private handleShortcuts(event: PluginEvent) {
+        let keyboardEvent = (event as PluginDomEvent).rawEvent as KeyboardEvent;
```

Comment:
nit: const. Also, can event be undefined or null at any point?

Reply:
looking at attachDomEvent and DefaultShortcut, no